readme.textile

```diff
@@ -6,6 +6,26 @@ the original version as well as the discussion of the game can be found "here":h
 Copyright В© the original code and sounds were developed by "Carlos Macintosh":http://twitter.com/KeyWasFull.
 You can find out how to play the game by reading _user guide.txt_ file.
 
+h2. Changes made by Brogar2000 on 04/16/2020
+* Made a data folder and moved a few files to it. Hopefully I can move more files over to it at a later date.
+* Renamed ss.bgt to game.bgt.
+* The level 5 boss works propperly now. I did not make any changes to fix this blitch, it seems to have fixed itself.
+* Sarcastic messages when you get killed have made a come back.
+* Changed some of the sounds, mainly some of the weapon and menu sounds. There are other sounds that have been changed as well.
+* Now using the DFSpeech/DLG method of speaking text. This means that Battle Zone can now be played with Jaws or NVDA once again. DFSpeech was made by http://masonasons.me and found at https://github.com/cartertemm?tab=repositories
+* Took out the old menu system and replaced it with m_pro, made by http://masonasons.me and found at https://github.com/cartertemm?tab=repositories
+* Battle Zone now makes use of dialogues. To scroll through dialogues, just press the enter key.
+* Battle Zone now has support for multiple languages. To make a new language, go into the languages folder. You can use the English file as a template. Remember not to add a file extention to your language file, or else the game will read out .txt at the end of the language name.
+* New language: Spanish, (Google Translated).

```

Comment:
Aha, that's why! For a Google translation it's not bad at all!

Reply:
Ah, yes. If someone who knows Spanish can fix the translation errors when I submit the second pull request, that would be wonderful, as I don't know Spanish very well. I only know English.